### PR TITLE
Adding update messages for the CLI v1.3.5-v1.4.4.

### PR DESCRIPTION
**Pull request by henvic**

### CODE REVIEW

File: node/electric/src/pages/updates/cli/index.md [hunk 1]
```diff
@@ -1,8 +1,31 @@
 ---
+<!--
```
Comment (jonnilundy) [inline node/electric/src/pages/updates/cli/index.md]: Is there a reason why you added this?

File: node/electric/src/pages/updates/cli/index.md [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+This file is autogenerated, do not edit; changes may be undone by the next 'wedeploy/cli: make release-notes-page'.
+-->
 title: "CLI"
-description: "Check out the latest releases of the WeDeploy"
+description: "Check out the release notes of latest releases of the WeDeploy CLI"
```
Comment (jonnilundy) [inline node/electric/src/pages/updates/cli/index.md]: Let's not change this until we can agree on a pattern for all the Update pages.

File: node/electric/src/pages/updates/cli/index.md [hunk 3]
```diff
@@ -1,8 +1,31 @@
...
 layout: "updates"
 updates:
+ -
+  version: 1.4.4
+  date: Mar 30th, 2018
+  description: Adding "we shell" command.
```
Comment (jonnilundy) [inline node/electric/src/pages/updates/cli/index.md]: Previously, you used past tense verbs in your descriptions (see: https://wedeploy.com/updates/cli/). I think this makes more sense from an English perspective.

File: node/electric/src/pages/updates/cli/index.md (hunk 1 above)
Comment (henvic) [reply node/electric/src/pages/updates/cli/index.md]: Yes. I'm auto-generating this page with the same release notes I am going to show the user itself after CLI updates. So I'd like to add a comment here, so it's less likely to be manually edited. Whoever wants to edit this, should edit https://github.com/wedeploy/cli/blob/master/update/releasenotes/page/page.go instead.

I am not sure if this is fine with our page generator, but if it is not I can just remove it.

Comment (jonnilundy) [reply node/electric/src/pages/updates/cli/index.md]: I see. That is great, but I think this snippet is breaking the generator so let's try removing. Thanks!

File: node/electric/src/pages/updates/cli/index.md [hunk 4]
```diff
@@ -1,8 +1,35 @@
 ---
+<!--
+This file is autogenerated, do not edit; changes may be undone by the next 'wedeploy/cli: make release-notes-page'.
+-->
 title: "CLI"
-description: "Check out the latest releases of the WeDeploy"
+description: "Check out the latest releases of the WeDeploy CLI"
```
Comment (jonnilundy) [inline node/electric/src/pages/updates/cli/index.md]: Is this necessary since the CLI title above it?

**Pull request merged**